iota-client/src/client.rs

```diff
@@ -179,6 +179,8 @@ pub enum Api {
     GetTips,
     /// `post_message` API
     PostMessage,
+    /// `post_message` API with remote pow
+    PostMessageWithRemotePow,
```

Comment:
should also add this to the `Api` type declared on `bindings/node/lib/index.d.ts`